### DISCUSSION TITLE
The Elastic constant versus pressure is not smooth.

### DISCUSSION THREAD

**Discussion by phys-chem** [deepmodeling/deepmd-kit#3463] [Q&A]
The training set involves structures over 150 GPa. Virial is not trained for my model. The Elastic constant fluctuates with pressure. Does anyone meet the same problem? Could you give me some suggestion? Thank you very much!
![image](https://github.com/deepmodeling/deepmd-kit/assets/62685413/1d96fc4d-8a14-4c02-bf1d-428043d1aa6d)

**Comment by wanghan-iapcm** [comment]
It is highly recommended to train with virial, because the elastic modulus are 2nd order derivatives of energy with cell.

**Reply by phys-chem** [reply]
I will try to train the virial.

**Comment by chazeon** [comment]
Even if the potential is well-trained, a strain of `1e-7` is too small.

**Reply by phys-chem** [reply]
Thank you! I have tried a larger strain but it is still not smooth.
